exist/windows.md

```diff
@@ -0,0 +1,128 @@
+
+If you moved partitions around, verify your Windows installation is still functioning before proceeding. If you cannot boot into Windows, restore your backup and try again using a reputable partition manager.
+
+![Windows after partitions](../images/ex-os/after_windows_disk_mgmt.png)
+
+For the next section, you will need your OpenCore USB ready to boot into the macOS installation. 
+
+## Reformat partition to APFS using Disk Utility
+
+It is advised to use [LauncherOption=Full](https://dortania.github.io/OpenCore-Post-Install/multiboot/bootstrap.html) or to create a custom BIOS entry for the USB so that during the installation process when the system reboots, it will not automatically boot into Windows. When system reboot does happen during the installation process, even if OpenCore loads, the picker's default option might be Windows so watch it carefully and be ready to press the arrow keys depending on how short the timeout is set in your config.plist: `Misc > Boot > Timeout`.
+
+![Mac install screen](../images/ex-os/mac_disk_util.png)
+
+Once you reach the main installation menu click Disk Utility and select the target partition. Click Erase in the top button bar to format the partition to APFS (or whatever you prefer).
+
+![Mac disk utility](../images/ex-os/mac_erase_button.png)
+
+![Mac format partition](../images/ex-os/mac_erase_partition.png)
+
+## Continue with macOS installation
+
+![Mac continue install](../images/ex-os/mac_install.png)
+
+Once installation is finished remove OpenCore USB and the system will boot into Windows.
+
+## Mount EFI partition from Windows
+
+Using a partition manager or diskpart, mount the EFI partition. [Using MiniTool Partition Wizard](https://www.partitionwizard.com/free-partition-manager.html), this is as simple as assigning a letter to the drive. It will then be visible to Windows in File Explorer.
+
+![Windows mount efi 1](../images/ex-os/mount_efi_windows_a.png)
+
+![Windows mount efi 2](../images/ex-os/mount_efi_windows_b.png)
+
+Plug your OpenCore USB in and copy contents of Open Core USB EFI into the mounted EFI partition. In File Explorer, you can see the EFI partition but will not be able to access it. An easy workaround is to just use [Explorer++](https://explorerplusplus.com/download). Right click and select Run as administrator to read and write to the EFI folder.
+
+![Windows explorer++](../images/ex-os/explorer++_windows.png)
+
+Backup your existing EFI folder before continuing. Copy the BOOT and OC folder from inside the USB EFI into the partition's EFI folder from within Explorer++. When prompted, overwrite the BOOTx64.efi that exists with the one from OpenCore.
+
+Open command prompt with Administrator priveleges and run:
+
+```
+bcdedit /set {bootmgr} path \EFI\OC\OpenCore.efi 
+```
+
+After this is complete, do a reboot of the system.  If the OpenCore picker shows, then you can skip the next section.
```

Comment:
Add this to the next section and remove it from this part.

Reply:
Applied